ui/appui-abstract/src/appui-abstract/UiItemsManager.ts

```diff
@@ -65,15 +68,15 @@ export type UiItemProviderOverrides = MarkRequired<AllowedUiItemProviderOverride
   MarkRequired<AllowedUiItemProviderOverrides, "providerId" | "stageUsages" | "stageIds">;                  // eslint-disable-line @typescript-eslint/indent
 
 /** Interface that defines an instance of a UiItemsProvider and its application specified overrides.
- *  @beta
+ * @beta
```

Comment:
Should this be deprecated too since the UiItemsProvider is?

Reply:
I'll just get rid of this tag, since the interface itself is not exported (internal type to the module).

Reply:
Tag removed.